images/linux/scripts/helpers/install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+################################################################################
+##  File:  install-helpers.sh
+##  Desc:  Helper functions for installing tools
+################################################################################
+
+function TestIsXenial
+{
+    lsb_release -d|grep 'Ubuntu 16' > /dev/null
+}
+
+function TestIsBionic
+{
+    lsb_release -d|grep 'Ubuntu 18' > /dev/null
+}
```

Comment:
I think it is better to have the following functions:
1) `isUbuntu16` (rename from `TestIsXenial`)
2) `isUbuntu18` (rename from `TestIsBionic`)
3) `getOSVersionLabel` that will return `xenial` or `bionic`